webextension/popup-events.js

```diff
@@ -219,97 +311,103 @@ document.addEventListener(
             }
           }
           break;
+        case ESCAPE_KEY:
+          // close popup
+          if (chrome && chrome.tabs) {
+            chrome.tabs.query({ active: true, currentWindow: true }, tabs => {
+              if (tabs && tabs.length > 0) {
+                sendMessageToTab(
+                  tabs[0].id,
+                  { action: "closePopup" },
+                  response => {
+                  }
+                );
+              }
+            });
+          } else {
+            chrome.runtime
+              .sendMessage({
+                action: "getActiveTab"
+              })
+              .then(
+                response => {
+                  if (response && response.tabs && response.tabs.length > 0) {
+                    sendMessageToTab(
+                      response.tabs[0].id,
+                      {
+                        action: "closePopup"
+                      },
+                      result => {
+                      }
+                    );
+                  }
+                },
+                error => {
+                  if (error) {
+                  }
```

Comment:
Should this really be ignored without logging? If so, please add a comment there explaining why.

Reply:
okay, I could call `Tools.logOnServer()` for this one.